tests/test_routedem.py

```diff
@@ -59,23 +59,6 @@ def _make_dem(target_path):
         dem_raster.SetGeoTransform(dem_geotransform)
         dem_raster = None
 
-    def test_routedem_invalid_algorithm(self):
-        """RouteDEM: fail when the algorithm isn't recognized."""
-        from natcap.invest import routedem
-        args = {
-            'workspace_dir': self.workspace_dir,
-            'algorithm': 'invalid',
-            'dem_path': os.path.join(self.workspace_dir, 'dem.tif'),
-            'results_suffix': 'foo',
-            'calculate_flow_direction': True,
-        }
-
-        RouteDEMTests._make_dem(args['dem_path'])
-        with self.assertRaises(RuntimeError) as cm:

```

Comment:
It looks like this test would still make sense if it checked for `KeyError` instead of `RuntimeError`. I think it's important that a bad algorithm raises some kind of error, so might as well keep this test around in my opinion.